model/action.go

```diff
@@ -69,7 +69,8 @@ type FunctionRef struct {
 	RefName string `json:"refName" validate:"required"`
 	// Function arguments
 	// TODO: validate it as required if function type is graphql
-	Arguments map[string]interface{} `json:"arguments,omitempty"`
+	// TODO map[string]interface{}
```

Comment:
Why we must define it as map[string]string？the value of arguments[key] may be an int or object

Reply:
`interface{}` don't play well with `deepCopy`. I think we can implement our type in this case (`IntOrObject`).

Reply:
That can work, will try tomorrow.

Reply:
see last commit